packages/generator/generators/app/templates/package.txt

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@kaizen/draft-<%= componentName %>",
+  "version": "1.4.1",
+  "description": "The draft <%= componentName %> component",
+  "scripts": {
+    "prepublish": "tsc --project tsconfig.dist.json",
+    "build": "yarn clean && yarn prepublish",
+    "build:watch": "yarn clean && yarn prepublish --watch",
+    "clean": "rimraf '*.d.ts' '*.js' '*.map' 'KaizenDraft/**/*.d.ts' 'KaizenDraft/**/*.js' 'KaizenDraft/**/*.map'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cultureamp/kaizen-design-system.git",
+    "directory": "draft-packages/<%= componentName %>"
+  },
+  "bugs": {
+    "url": "https://github.com/cultureamp/kaizen-design-system/issues"
+  },
+  "files": [
+    "**/*",
+    "!**/*.ts",
+    "!**/*.tsx",
+    "**/*.d.ts",
+    "!**/*.spec.*",
+    "!**/*.snap",
+    "!draft-templates",
+    "!stories",
+    "!tsconfig.dist.json"
+  ],
+  "author": "",
+  "private": false,
+  "license": "MIT",
+  "dependencies": {},
+  "peerDependencies": {
+    "@kaizen/design-tokens": "^1.2.0",
```

Comment:
Should we update this to the lastest (`^1.6.0`), or is this intentional to loosen the restrictions for the consuming package?